src/script_functions.c

```diff
@@ -158,7 +158,7 @@ int c_get_window_name(lua_State *lua)
 	}
 
 	WnckWindow *window = get_current_window();
-	char *test = window ? (char*)wnck_window_get_name(window) : "";
+	const char *test = window ? wnck_window_get_name(window) : "";
```

Comment:
We don't modify this, so why cast it instead of keeping the constness from the source and wanted by the next call?